final-project/scripts/main.js

```diff
@@ -1,10 +1,110 @@
 import "../styles/main.scss";
 
-// testing that `const` and arrow functions transpile correctly
-const before = "Hello -world!";
-const after = before
-  .split("")
-  .filter(char => char !== "-")
-  .join("");
-console.log({ before });
-console.log({ after });
+const url = "https://wt-4662f45b9eefda7172b747b28d23efdb-0.sandbox.auth0-extend.com/blog";
+var articlesIndex = 1;
+
+getArticlesFromUrl();
+
+async function getArticlesFromUrl() {
+  fetch(url).then(resp => resp.json()).then(function(data) {
+    loadCarousel(data.articles);
+  });
+}
+
+function loadCarousel(articles) {
+  articles.forEach((article, index) => {
+    var containerDiv = document.getElementById("carousel_container");
+
+    var descriptionContainer = document.createElement("div");
+    descriptionContainer.setAttribute("class", "section__six__carousel__cards__content__description" );
+    
+    var imageContainer = document.createElement("div");
```

Comment:
I liked that you used JS to assemble the article parts and you got used to `createElement` the only thing is that if the HTML or CSS changes you probably will need to make some changes to this part too. A better solution would be to "clone" an already existing HTML node with everything only changing the content.